invokeai/app/cli_app.py

```diff
@@ -162,8 +169,8 @@ def invoke_cli():
 
     while True:
         try:
-            cmd_input = input("> ")
-        except KeyboardInterrupt:
+            cmd_input = input("invoke> ")
```

Comment:
What if this showed the current node id instead (or in addition)? Might make it easier for users to link to previous nodes. I'd suggest adding the session id too, but that string is huge =/.

Reply:
I like that. How do I get the current node and its ID? Does it make sense to use the first 8 characters of the session ID?